examples/resources/pingfederate_oauth_client_registration_policy/resource.tf

```diff
@@ -0,0 +1,39 @@
+resource "pingfederate_oauth_client_registration_policy" "registrationPolicy" {
```

Comment:
I didn't find an  example for this one - this is the default configuration for this type when created in the UI

Reply:
This works